NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -519,4 +520,82 @@ class NeuroID private constructor(
     }
 
     fun getSDKVersion() = NIDVersion.getSDKVersion()
+
+    // new Session Commands
+    fun clearSessionVariables() {
+        userID = ""
+//        registeredUserID = ""
+    }
+
+    fun startSession(sessionID: String = ""): SessionStartResult {
+        if (clientKey == "") {
+            NIDLog.e(
+                msg = "Missing Client Key - please call configure prior to calling start"
+            )
+            return SessionStartResult(false, "")
+        }
+
+        if (userID != "" || isSDKStarted) {
+            stopSession()
+        }
```

Comment:
Tiny question, why would we stop the session if the sdk is started? Is it to prevent multiple sessions in progress?